GPU/GLES/GPU_GLES.cpp

```diff
@@ -186,6 +186,12 @@ u32 GPU_GLES::CheckGPUFeatures() const {
 		features |= GPU_USE_SINGLE_PASS_STEREO;
 	}
 
+	if (draw_->GetBugs().Has(Draw::Bugs::ADRENO_RESOURCE_DEADLOCK)) {
+		if (PSP_CoreParameter().compat.flags().OldAdrenoPixelDepthRoundingGL) {
+			features |= GPU_ROUND_FRAGMENT_DEPTH_TO_16BIT;
```

Comment:
`CheckGPUFeaturesLate()` might add another flag (like `GPU_ROUND_DEPTH_TO_16BIT` or `GPU_SCALE_DEPTH_FROM_24BIT_TO_16BIT`), not sure if this is really a problem or not, though.  In theory those flags can coexist...

Reply:
Right, I'll re-order it to do it after CheckGPUFeatureLate, and also check for vendor workarounds.